third_party/packages/cupertino_icons/example/example.md

```diff
@@ -0,0 +1,12 @@
+Please find the usage of [`cupertino_icons`](https://pub.dev/packages/cupertino_icons) which is used via Flutter's `CupertinoIcons` class below:
+
+```dart
```

Comment:
This needs to use code excerpts if it's going to contain actual code; see https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#readme-code for instructions on setting that up. (It's about README.md, but the same applies to example.md)

Reply:
Hi @stuartmorgan, I went through the doc. There is no test dart file in the package & the `cupertino_icons.dart` also contains no code. I am thinking of adding this example code in `cupertino_icons.dart` file.

> Ideally the code should be in one of the examples, but you could also put the code in a test.

Or should I add an example app but that would add a lot of boilerplate code. What would you suggest?

Reply:
> I am thinking of adding this example code in `cupertino_icons.dart` file.

This would cause the code to be in the build application of every app using the package, which we don't want.

I don't think there's any reason we couldn't put it in a test/ file; it wouldn't actually be useful as a test, but it would cause the example code to be linted and compiled so that we knew it was staying current, which is the point of the excerpt system, and running a Dart unit test in CI is extremely fast.

Reply:
I have updated the `example.md` file with code excerpts. I had to update `pubspec.yaml` as well because I had to add import `import 'package:flutter/cupertino.dart'` for writing test. Please review.